packages/tools/generator-widget/generators/app/templates/pluggable/native/fullTemplateTsFn/src/components/Badge.tsx.ejs

```diff
@@ -0,0 +1,37 @@
+import { ReactElement, createElement, ElementType } from "react";
+import { Platform, Text, TouchableNativeFeedback, TouchableOpacity, View } from "react-native";
+
+import { mergeNativeStyles } from "@mendix/pluggable-widgets-tools";
+
+import { BadgeStyle, defaultBadgeStyle } from "../ui/styles";
+
+export interface BadgeProps {
+    value: string;
+    style: BadgeStyle[];
+    onClick: () => void;
+}
+
+export function Badge(props: BadgeProps): ReactElement | null {
```

Comment:
Seems like the result of this function will never be null as you are always returning an element.